src/app/components/filebrowseruss/filebrowseruss.component.html

```diff
@@ -10,15 +10,17 @@
 -->
 
 <div style="height: 100%;">
-    <img src="../../../assets/explorer-uparrow.png" (click)="levelUp()" style="width: 20px; height: 20px; filter: brightness(3); cursor: pointer;">
+    <img src="../../../assets/explorer-uparrow.png" (click)="levelUp()" [style]="treeStyle">
     <!-- <svg width="14" height="16" viewBox="0 0 14 16" fill-rule="evenodd" (click)="levelUp()"><path d="M6 3.4V16h2V3.4l4.7 4.9L14 7 7 0 0 7l1.3 1.3z"></path></svg> -->
-    <div class="filebrowseruss-search">
-        <input [(ngModel)]="path" placeholder="Enter a directory..." class="filebrowseruss-search-input" (change)="displayTree(path, false)" [disabled]="isLoading">
+    <div class="filebrowseruss-search"  [ngStyle] = "searchStyle">
+        <input [(ngModel)]="path" placeholder="Enter a directory..." class="filebrowseruss-search-input" [ngStyle] = "inputStyle" (change)="displayTree(path, false)">
+        <!-- <input [(ngModel)]="path" placeholder="Enter a directory..." class="filebrowseruss-search-input" (change)="displayTree(path, false)" [disabled]="isLoading"> -->
```

Comment:
Update: https://github.com/zowe/zlux-file-explorer/pull/25/files

Jordan added a 2 sec timeout to his loadUserHomeDirectory logic. This may be suitable now